jmeter-service/eventhandler.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"errors"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	logger "github.com/sirupsen/logrus"
+	"net/url"
+)
+
+// EventHandler handles events of type 'test.triggered' and kicks off
+// the TestRunner to execute the JMeter tests
+type EventHandler struct {
+	testRunner *TestRunner
+}
+
+func (e *EventHandler) handleEvent(event cloudevents.Event) error {
+	var shkeptncontext string
+	if err := event.Context.ExtensionAs("shkeptncontext", &shkeptncontext); err != nil {
+		return err
+	}
+
+	data := &keptnv2.TestTriggeredEventData{}
+	if err := event.DataAs(data); err != nil {
+		logger.WithError(err).Error("Unable to decode 'test.triggered' event data")
+		return nil
+	}
+
+	if event.Type() != keptnv2.GetTriggeredEventType(keptnv2.TestTaskName) {
+		logger.Errorf("Received unexpected keptn event: %s", event.Type())
```

Comment:
I think we can switch to `Info` or `Warning`, as the reception of an unexpected event is arguably not an error in the jmeter-service